{"name": "登录失败", "status": "passed", "start": 1655543132103, "stop": 1655543132103, "uuid": "3ff8004c-b16e-447f-ab91-ba393998192b", "historyId": "cad4c8899b58c63b1ca84793cb05b499", "testCaseId": "76d8a45a420a86cfad6f84d07a44a7e3", "fullName": "TestCases.test_login.TestLogin#test_login_failed", "labels": [{"name": "story", "value": "测试登录"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-0ABI0I6V"}, {"name": "thread", "value": "17096-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}